index.html

```diff
@@ -261,7 +262,13 @@ <h2 class="contact-me">Contact me</h2>
   </fieldset>
 </div>
 </section>
+
+<!-- Popup Windows Start -->
+<section id="modal-popup">
```

Comment:
- [ ]  I can not scroll down and see all the pop elements as you can see on the video shared below ⬇️  Please make sure to fix that and note that your pop-up windows need to stick to the Figma design as much as possible in order to have a beautiful website and good user experience 👍. Your popup window can be improved, and your button also, and elements spacing(padding/margin) line height, and position are a bit different, please check the screenshots below ⬇️ to see the differences.  Please make sure that your mobile menu design is similar to the figma template as required for this project 👍

https://www.loom.com/share/1385e9b1996c419eadba6ea82c0bfc81